modules/persistent-storage-fibre-provisioning.adoc

```diff
@@ -32,11 +32,13 @@ spec:
   accessModes:
     - ReadWriteOnce
   fc:
-    targetWWNs: ['500a0981891b8dc5', '500a0981991b8dc5'] <1>
-    lun: 2
+    wwids: [scsi-3600508b400105e210000900000490000] <1>
+    targetWWNs: ['500a0981891b8dc5', '500a0981991b8dc5'] <2>
+    lun: 2 <2>
     fsType: ext4
 ----
-<1> Fibre Channel WWNs are identified as
+<1> World wide identifiers (WWIDs). Either FC `*wwids*` or a combination of FC `*targetWWNs*` and `*lun*` must be set, but not both simultaneously. The FC WWID identifier is recommended over the WWNs target because it is guaranteed to be unique for every storage device, and independent of the path that is used to access the device. The WWID identifier can be obtained by issuing a SCSI Inquiry to retrieve the Device Identification Vital Product Data (`*page 0x83*`) or Unit Serial Number (`*page 0x80*`). FC WWIDs are identified as `*/dev/disk/by-id/*` to reference the data on the disk, even if the path to the device changes and even when accessing the device from different systems.
```

Comment:
This is a lot of information to pack in to a callout. Did you consider other ways to add this information?
Here I see a key definition, a key usage rule, a recommendation, a procedure about how to obtain the key value, and so on. 

Regarding specifics: 

FC `*wwids*`: I don't think we need bold text here. And can we get rid of the "FC"? The key name is sufficient.

"but not both simultaneously" : I'm not certain what "both" refers to here. 

"WWID identifier": ~~identifier~~. It's already in the name!

"The WWID identifier can be obtained by issuing a SCSI Inquiry to retrieve the Device Identification Vital Product Data (`*page 0x83*`) or Unit Serial Number (`*page 0x80*`)."
This hints at a way to get this, but it's not all there.